vnext/Shared/CreateModules.h

```diff
@@ -10,7 +10,7 @@
 namespace facebook {
 namespace react {
 
-class MessageQueueueThread;
```

Comment:
`MessageQueueueThread`?

Reply:
one two many "ue"s :)
class MessageQueueThread;

Reply:
Which made me think this was just noise and could be removed.
Curiously enough, if I remove the correct name `MessageQueueThread`, compilation fails.

Reply:
I don't see any reference to MessageQueueueThread though other than this forward declaration though. What error do you get?

Reply:
Nevermind. It builds fine now without the forward declaration.